### ISSUE TITLE
process undefined

### ISSUE THREAD

**Issue by corymickelson** [skatejs/skatejs#602]
Uncaught ReferenceError: process is not defined. Thrown here: 

```javascript 
   if (typeof process === 'undefined') {
      /* eslint no-undef: 0 */
      process = { env: { NODE_ENV: 'production' } };
    }
```

just trying to run the example at the top of the docs.

**Comment by treshugart** [comment]
I was unable to reproduce using the `1.0.0-beta.1` in JSBin (how were you doing it?), but I removed that check within the `vdom` module anyways as it shouldn't be necessary anymore for incremental DOM. It's up on master. Let us know if it's still an issue.

**Comment by treshugart** [comment]
Just released `1.0.0-beta.2` to NPM.
